runtime/gc_glue_java/MarkingSchemeRootClearer.cpp

```diff
@@ -329,6 +330,26 @@ MM_MarkingSchemeRootClearer::scanContinuationObjects(MM_EnvironmentBase *env)
 						while (NULL != object) {
 							gcEnv->_markJavaStats._continuationCandidates += 1;
 							omrobjectptr_t next = _extensions->accessBarrier->getContinuationLink(object);
+
```

Comment:
We should put this block within

#if defined(OMR_GC_CONCURRENT_SCAVENGER)

We do it for other CS aware code in Marking (for example in OMR's MM_MarkingScheme::fixupForwardedSlot)

Still thinking if we should add any runtime check... @dmitripivkine, an opinion?